tests/realtime/realtimeUtils.js

```diff
@@ -16,51 +16,51 @@ const tu = require('../testUtils');
 const u = require('./utils');
 
 describe('tests/realtime/realtimeUtils.js, realtime utils Tests >', () => {
+  const rootSubjNA = tu.namePrefix + 'NA';
+  const rootSubjNAUS = rootSubjNA + '.US';
+  const rootSubjNAUSCA = rootSubjNA + '.CA';
   const newSubject = {
-    absolutePath: 'NA.US.CA.SF',
+    absolutePath: rootSubjNAUSCA + '.SF',
     name: 'SF',
   };
   const updatedSubject = {
     new: {
-      absolutePath: 'NA.US.CA.SF',
+      absolutePath: rootSubjNAUSCA + '.SF',
       name: 'SF',
       helpurl: 'helpme.com',
     },
     old: {
-      absolutePath: 'NA.US.CA.SF',
+      absolutePath: rootSubjNAUSCA + '.SF',
       name: 'SF',
       helpurl: '',
     },
   };
+  const aspectOne = {
+    name: tu.namePrefix + 'temperature',
+    timeout: '30s',
+    isPublished: true,
+    tags: ['temp'],
+  };
   const looksLikeSampleObjNA = {
     value: '10',
-    name: 'NA|temperature',
-    absolutePath: 'NA',
+    name: rootSubjNA + '|' + aspectOne.name,
```

Comment:
create aspect and subject to test query case insensitivity.
add test prefix to aspect and subject.